src/api.js

```diff
@@ -7,6 +7,10 @@ async function get(path, params = {}) {
 }
 
 function getCompare(hostIds = []) {
+    if (!Array.isArray(hostIds)) {
+        hostIds = [ hostIds ];
+    }
```

Comment:
@beav The bug you hit is if a single host_id is passed, the query string parser interprets this as a single value case and converts it to a string rather than an array. This passes it as `host_ids` instead of `host_ids[]` to the backend. This ensures it's always an array and get's converted properly by the params.